### DISCUSSION TITLE
Thank you so much!

### DISCUSSION THREAD

**Discussion by ghost** [Klocman/Bulk-Crap-Uninstaller#570] [General]
Hello developer, thank you for putting your time and sweat into this thing! it just uninstalled a lot of crap and leftovers from my laptop, made the reboot process very fast. It is almost half the size of the big uninstallers out there but has all the saftey feature packed to let you select what you want to delete. I just want to thank you for awesome development and blessings! Keep it running, this is most important tool in my arsenal.

**Comment by Klocman** [comment]
Thanks!
